gdx/src/com/badlogic/gdx/scenes/scene2d/utils/TiledDrawable.java

```diff
@@ -36,6 +39,8 @@ public TiledDrawable (TextureRegionDrawable drawable) {
 	}
 
 	public void draw (Batch batch, float x, float y, float width, float height) {
+		float batchColor = batch.getPackedColor();
+		batch.setColor(tint);

```

Comment:
A proper tint would multiply the batch color by the drawable color. Eg, this is what SpriteDrawable does.